extensions/amp-ad-network-doubleclick-impl/0.1/doubleclick-a4a-config.js

```diff
@@ -53,18 +54,30 @@ const DOUBLECLICK_A4A_EXPERIMENT_NAME = 'expDoubleclickA4A';
 // can disable profiling again, we can return these constants to being
 // private to this file.
 /** @const {!../../../ads/google/a4a/traffic-experiments.ExperimentInfo} */
-export const DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES = {
```

Comment:
I see we export the value, is `DOUBLECLICK_A4A_EXTERNAL_EXPERIMENT_BRANCHES` used somewhere else that we need to change naming?